acceptance/acceptance_test.go

```diff
@@ -1289,31 +1276,91 @@ func testAcceptance(
 								h.AssertEq(t, output, expectedOutput)
 							}
 						})
+
+						when("builder is untrusted", func() {
+							it("uses the 5 phases", func() {
+								var buf bytes.Buffer
+								var cmd *exec.Cmd
+								cmd = subjectPack(
+									"build", repoName,
+									"--builder", builderName, // untrusted by default
+									"-p", filepath.Join("testdata", "mock_app"),
+									"--publish",
+									"--network", "host",
+								)
+
+								cmd.Stdout = &buf
+								cmd.Stderr = &buf
+
+								h.AssertNil(t, cmd.Start())
+
+								go terminateAtStep(t, cmd, &buf, "[detector]")
```

Comment:
Canceling out early to avoid adding too much time to the acceptance tests. But maybe it makes sense to let this one go all the way through, so that we have verification that the 5 phases build successfully (this is implicitly covered in the test above since `--publish` is used without `--trust-builder` - but maybe it is better to be explicit.

Reply:
I think it's worth checking the 5 phases. If you don't want to add time to the acceptance tests, maybe you can just add the checks to the previous test.